src/whylogs/app/writers.py

```diff
@@ -371,7 +371,7 @@ def write(self, profile: DatasetProfile, rotation_suffix: str = None):
         """
         Write a dataset profile to WhyLabs
         """
-        t = async_wrap(self._write_protobuf, profile, rotation_suffix)
+        t = async_wrap(self._write_protobuf, profile)
```

Comment:
WhyLabs Writer does not require or take in rotation_suffix param, so i removed it here.